ewapt.py

```diff
@@ -983,240 +983,6 @@ def check(str):
 		response = "One door at a time, please."
 		return await ewutils.send_message(cmd.client, cmd.message.author, ewutils.formatMessage(cmd.message.author, response))
 
-# TODO: Remove after Double Halloween
-async def trickortreat(cmd = None):
```

Comment:
should we really delete all this code? I think it'd make sense to keep it around for next year. just remove the commands from the command map. this goes for all the double halloween commands

Reply:
Yeah, that sounds more reasonable. I forgot that a command is essentially removed from the game if it's not on the command map anyways.